sysutils/pfSense-pkg-System_Patches/files/usr/local/www/system_patches.php

```diff
@@ -48,28 +48,38 @@
 	}
 }
 
-if (($_POST['act'] == "fetch") && ($a_patches[$_POST['id']])) {
-	$savemsg = patch_fetch($a_patches[$_POST['id']]) ? gettext("Patch Fetched Successfully") : gettext("Patch Fetch Failed");
-}
-if (($_POST['act'] == "test") && ($a_patches[$_POST['id']])) {
-	$savemsg = patch_test_apply($a_patches[$_POST['id']]) ? gettext("Patch can be applied cleanly") : gettext("Patch can NOT be applied cleanly");
-	$savemsg .= " (<a href=\"system_patches.php?id={$_POST['id']}&amp;fulltest=apply\" usepost>" . gettext("detail") . "</a>)";
-	$savemsg .= empty($savemsg) ? "" : "<br/>";
-	$savemsg .= patch_test_revert($a_patches[$_POST['id']]) ? gettext("Patch can be reverted cleanly") : gettext("Patch can NOT be reverted cleanly");
-	$savemsg .= " (<a href=\"system_patches.php?id={$_POST['id']}&amp;fulltest=revert\" usepost>" . gettext("detail") . "</a>)";
-}
-if (($_POST['fulltest']) && ($a_patches[$_POST['id']])) {
-	if ($_POST['fulltest'] == "apply") {
-		$fulldetail = patch_test_apply($a_patches[$_POST['id']], true);
-	} elseif ($_POST['fulltest'] == "revert") {
-		$fulldetail = patch_test_revert($a_patches[$_POST['id']], true);
+if ($a_patches[$_POST['id']]) {
+	$savemsg = gettext("Patch ");
+	$descr = $a_patches[$_POST['id']]['descr'];
```

Comment:
What about previously saved patches that have no description?

This should probably log the patch `uniqid` and then also print the description, if present. The patch ID is shown on the page for every patch already.